src/main/resources/db/migration/V202409221730__create_feedback_and_feedback_response_tables.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS feedback (
+                                        id BIGSERIAL PRIMARY KEY,
+                                        company_id BIGINT NOT NULL,
+                                        sender_id BIGINT NOT NULL,
+                                        text TEXT NOT NULL,
+                                        is_anonymous BOOLEAN DEFAULT FALSE,
+                                        created_at DATE DEFAULT CURRENT_DATE,
+                                        status VARCHAR(50) DEFAULT 'Unreviewed'
```

Comment:
You may want to consider adding modification date, so you can tell when was the last time it was modified. you probably didn't include that in your document, worth thinking about it